cmd/bootstrapper/initialize_kubelet.go

```diff
@@ -24,6 +24,10 @@ var (
 			if err != nil {
 				return err
 			}
+			err = cmd.MarkPersistentFlagRequired("cluster-dns-ip")
+			if err != nil {
+				return err
+			}
```

Comment:
You're marking it as required here, but I don't think it should be considering it wasn't needed in the past. Not setting it is valid, in that it will use the default value decided by the kubelet

Reply:
Valid point; which I considered. 

The default value decided by the kubelet  is :[the host's DNS servers](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/)